packages/react-vapor/src/components/editor/tests/codeEditorSelectors.spec.ts

```diff
@@ -0,0 +1,21 @@
+import {IReactVaporState} from '../../../ReactVapor';
+import {CodeEditorState} from '../CodeEditorReducers';
+import {CodeEditorSelectors} from '../CodeEditorSelectors';
+
+describe('CodeEditorSelectors', () => {
+    describe('getValue', () => {
+        it('should not throw and return an empty string when passing a falsy id', () => {
```

Comment:
```suggestion
        it('should return an empty string when passing a falsy id', () => {
```
nitpick but since you return an empty string, we know it does not throw :yum: